sphinx/tutorial/rose/configurations.rst

```diff
@@ -93,7 +90,7 @@ For example:
 
 .. hint::
 
-   In Rose configuration files settings do not normally require quotation.
+   In Rose configuration files, settings do not normally require quotation.
```

Comment:
No longer true - settings must be a valid Python or Jinja2 literal!

Reply:
A slightly broader matter, perhaps for another PR, but not if you agree and wish to do it: I think that "Why Use Rose Configurations" should be near the top of this file.

Reply:
> No longer true - settings must be a valid Python or Jinja2 literal!

I think that only applies to `[template variables]` settings?

Reply:
You are right, but since that it > 50% of use-cases I think that this is unhelpful